web-repl/js/wasm-worker.js

```diff
@@ -0,0 +1,47 @@
+import { threads } from 'wasm-feature-detect';
+import * as Comlink from 'comlink';
+
+var replInstance;
+// Wrap wasm-bindgen exports (the `generate` function) to add time measurement.
+function wrapExports({ Repl }) {
+  replInstance = new Repl();
+  return ({ textContent }) => {
+    try {
+      return replInstance.execute_lurk(textContent);
+    } catch (err) {
+      console.log('An error has occured, resetting engine state: ' + err);
+      replInstance = new Repl();
+      return JSON.stringify({iterations: 0, result: 'An error has occured, resetting engine state: ' + err});
+    }
+  };
+}
+
+async function initHandlers() {
+  let [singleThread, multiThread] = await Promise.all([
+    (async () => {
+      const singleThread = await import('../pkg/index.js');
```

Comment:
What's the rationale for importing `index.js` instead of `web_repl.js` like in the wasm-bindgen-rayon demo?

Reply:
There doesn't seem to be a generated `web_repl.js` in the `pkg` folder, only `index.js`

Reply:
Ah okay, I had assumed there was one based on my earlier experiments.

Reply:
I wonder if this is what is causing the circular dep?